data/json/items/armor/ballistic_armor.json

```diff
@@ -177,13 +186,14 @@
     "name": { "str": "heavy steel ballistic plate" },
     "repairs_like": "hsurvivor_jumpsuit",
     "//": "based on level 3 plates. Discuss high temp quenching so gonna assume tempered steel",
-    "//2": "link like: https://www.shindn.com/products/nij-iiia?variant=35614363582627&currency=USD&utm_medium=product_sync&utm_source=google&utm_content=sag_organic&utm_campaign=sag_organic&utm_campaign=gs-2019-08-03&utm_source=google&utm_medium=smart_campaign&gclid=Cj0KCQjw5-WRBhCKARIsAAId9FlGQAManRGZ0qpV9AIkD23FZfpm4aPDI4PHE9qPktZ2bdQY2NmcIVkaAs2pEALw_wcB",
+    "//2": "link like: https://www.spartanarmorsystems.com/spartan-omega-ar500-body-armor-single-plate/",
     "description": "A heavy concave ballistic plate made of tempered steel, designed to stop small-caliber rounds.  \"NIJ III PROTECTION\" is etched into the back of the plate.  This is intended to be inserted into a ballistic vest, and can't be worn on its own.",
-    "weight": "2500 g",
-    "//3": "30cm x 25cm x .55cm",
-    "volume": "413 ml",
+    "weight": "3912 g",
+    "//3": "25.4cm x 30.48cm x 0.635cm plate measurements (thin plate)",
```

Comment:
This one seems more reasonable as a reference but you would need to update the steel to be the right width and remove the nylon since these aren't wrapped. 

That would put there overall protection at 57, 57, 31.75 which seems fine and still puts them in the right ballpark.

Reply:
Adjusted the plate thickness to 6.35mm as you asked.